doc/releases/changelog-dev.md

```diff
@@ -151,13 +151,25 @@
         return qml.expval(sum_op)
   ```
 
-  ```
+  ```python
```

Comment:
In this case we should have `pycon` since this section of the code is representing a console output. 

```suggestion
  ```pycon
```